src/components/OrgSettings/AgendaItemCategories/Preview/AgendaCategoryPreviewModal.test.tsx

```
@@ -0,0 +1,122 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import AgendaCategoryPreviewModal from './AgendaCategoryPreviewModal';
+
+// Mock translation function
+const mockT = (key: string) => key;
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)